OutOfSchool/OutOfSchool.EmailSender/EmailSenderService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Quartz;
+using Quartz.Impl;
+
+namespace OutOfSchool.EmailSender;
+
+public class EmailSenderService : IEmailSenderService
+{
+    private readonly IScheduler scheduler;
+
+    public EmailSenderService()
+    {
+        scheduler = new StdSchedulerFactory().GetScheduler().Result;
+        scheduler.Start().Wait();
+    }
+
+    public async Task SendAsync(string email, string subject, (string html, string plain) content, DateTime? expirationTime = null)
+    {
+        expirationTime ??= DateTime.MaxValue;
+
+        var jobData = new JobDataMap
+        {
+            { "Email", email },
+            { "Subject", subject },
+            { "HtmlContent", EncodeToBase64(content.html) },
+            { "PlainContent", EncodeToBase64(content.plain) },
+            { "ExpirationTime", expirationTime },
```

Comment:
(Code style) These 5 string literals are used here and in EmailSenderJob. Probably they could be introduced as string constants?